ts/IterableInbox.tsx

```diff
@@ -26,22 +24,42 @@ import IterableInboxCustomizations from './IterableInboxCustomizations'
 const RNIterableAPI = NativeModules.RNIterableAPI
 const RNEventEmitter = new NativeEventEmitter(RNIterableAPI)
 
-const SCREEN_WIDTH = Dimensions.get('window').width
+import useDeviceOrientation from './useDeviceOrientation'
 
 type inboxProps = {
    messageListItemLayout: Function,
-   customizations: IterableInboxCustomizations
+   customizations: IterableInboxCustomizations,
+   tabBarHeight: number,
+   tabBarPadding: number
 }
 
-const IterableInbox = ({ messageListItemLayout, customizations }: inboxProps) => {
+const IterableInbox = ({
+   messageListItemLayout, 
+   customizations,
+   tabBarHeight,
+   tabBarPadding
+}: inboxProps) => {
    const defaultInboxTitle = "Inbox"
    const [selectedRowViewModelIdx, setSelectedRowViewModelIdx] = useState<number>(0)
    const [rowViewModels, setRowViewModels] = useState<InboxRowViewModel[]>([])
    const [loading, setLoading] = useState<boolean>(true)
    const inboxDataModel = new IterableInboxDataModel()
-
    const [animatedValue, setAnimatedValue] = useState<any>(new Animated.Value(0))
 
+   let { width, height, isPortrait } = useDeviceOrientation()
```

Comment:
Are these backwards? In `useDeviceOrientation`, it's `height` first.

Reply:
Yes, good catch my bad. I will change that.